src/main/java/de/kekru/struktogrammeditor/control/GlobalSettings.java

```diff
@@ -67,25 +69,27 @@
 		readBuildInfoFile();
 	}
 
-	public static void init(){
-		for(int i=0; i < EinstellungsDialog.anzahlStruktogrammElemente; i++){
-			elementBeschriftungenZumEinfuegenInDasStruktogramm[i] = EinstellungsDialog.standardWerte[i];
+	public static void init() {
+		for (int i = 0; i < EinstellungsDialog.anzahlStruktogrammElemente; i++){
+			elementBeschriftungenZumEinfuegenInDasStruktogramm.add(EinstellungsDialog.standardWerte[i]);
 		}
 		
 		loadSettings();
 	}
 	
-	private static void readBuildInfoFile(){
-
+	private static void readBuildInfoFile () {
 		try {
-	
 			Properties pr = new Properties();
-
 			InputStream in = null;
 			try {
-				in = new BufferedInputStream(GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE));
-				pr.load(in);
-				in.close();
+				if (GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE) == null) {
+					System.err.println("Error no Buildinfo file could be found");
+				} else {
+					in = new BufferedInputStream(GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE));
+					pr.load(in);
+					in.close();
+					return;
+				}
 			} catch (IOException e) {
 				e.printStackTrace();
 			}

```

Comment:
Having Java 8 now, we should use try-with-ressources. then no close() is required.  
Also avoid using printstackTrace(). Better just throw a RuntimeException.  
And the BUILDINFO_FILE should always be present, when your revert pom.xml

```suggestion
try (InputStream in = new BufferedInputStream(GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE))) {
  pr.load(in);
  return;
} catch (IOException e) {
  throw new RuntimeException(e);
}			
```